pkgs/development/python-modules/numba/default.nix

```diff
@@ -104,6 +112,13 @@ in buildPythonPackage rec {
     "numba"
   ];
 
+  passthru.testers.cuda-detect =
```

Comment:
why not put this in `passthru.tests`?

Reply:
Because a successful build of this derivation does not indicate a passing test. This derivation builds a script that can be run outside the sandbox to perform the test.

Here's a relevant conversation, and it would really great to get some inputs from you: https://matrix.to/#/%23cuda%3Anixos.org/%24Uf81HxHEutaiFClu7LOCY-FiWg6DkO-KPxSlTP-inWQ?via=someonex.net&via=matrix.org&via=nixos.dev&via=amber.tel

Reply:
Ah gotcha, thanks for sending this chat! I don't always check matrix... I see the issue now -- i think it's fine to do whatever here and leave a little comment explaining the situation